packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.light.scss

```diff
@@ -0,0 +1,20 @@
+gux-form-field-color {
+  input {
+    &:focus,
+    &:focus-visible {
```

Comment:
This focus styling here will prevent 2 focus rings from rendering - we only want one focus ring on the outer container to render.